test/peering-test.js

```diff
@@ -195,6 +200,14 @@ describe('Peering', function() {
     });
 
     it('should gossip blocks over standard and brontide', async () => {
+      const seen = {
```

Comment:
Nit: removal of previous comment helping to clarify what this object is used for, since you can't see the whole test in a single screen.